app-misc/qstamina/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
There are missing information. Please take a look at https://github.com/gentoo/gentoo/blob/master/sys-libs/argp-standalone/metadata.xml for example.